recipes/opengv/all/conanfile.py

```diff
@@ -1,10 +1,12 @@
-from conans import ConanFile, CMake, tools
-from conans.errors import ConanInvalidConfiguration
 import os
 import textwrap
 
-required_conan_version = ">=1.43.0"
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.cmake import CMakeToolchain, CMake, cmake_layout, CMakeDeps
+from conan.tools.files import get, copy, rmdir, save, apply_conandata_patches, export_conandata_patches
 
+required_conan_version = ">=1.43.0"
```

Comment:
Is this adecuate?

Reply:
def not `export_conandata_patches` is super new

```suggestion
required_conan_version = ">=1.52.0"
```